### PR TITLE
fix: action selector popup value for integrations

### PR DESCRIPTION
**Pull request by ravikp7**
## Description
This fixes an issue where the action selector value was not showing up in the popup for integration options. This issue was introduced in release by [this PR](https://github.com/appsmithorg/appsmith/pull/23242).

Before|After
--|--
Screenshot 2023-05-22 at 11 28 54 PM|Screenshot 2023-05-22 at 11 28 26 PM

#### Type of change
- Bug fix (non-breaking change which fixes an issue)

- [ ] Manual
- [ ] Jest
- [x] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Test-plan-implementation#speedbreaker-features-to-consider-for-every-change) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans/_edit#areas-of-interest)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by rimildeyjsr — APPROVED** [review]
LGTM

File: app/client/cypress/support/Pages/PropertyPane.ts [hunk 1]
```diff
@@ -429,4 +429,10 @@ export class PropertyPane {
       "be.disabled",
     );
   }
+
+  public AssertSelectValue(value: string) {
+    cy.xpath(`//button[contains(@class, 't--open-dropdown-${value}')]`).should(
```
Comment (Aishwarya-U-R) [inline app/client/cypress/support/Pages/PropertyPane.ts:434]: Create a locator variable & use it here. No direct locators.

Comment (ravikp7) [reply app/client/cypress/support/Pages/PropertyPane.ts]: Missed somehow, changed.

File: app/client/cypress/support/Pages/PropertyPane.ts [hunk 2]
```diff
@@ -429,4 +429,8 @@ export class PropertyPane {
       "be.disabled",
     );
   }
+
+  public AssertSelectValue(value: string) {
+    cy.xpath(this.locator._selectByValue(value)).should("exist");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/support/Pages/PropertyPane.ts:434]: Use `agHelper.AssertElementExist()`

Comment (Aishwarya-U-R) [reply app/client/cypress/support/Pages/PropertyPane.ts:434]: Use helpers as much as possible avoiding direct cy statements.

Comment (ravikp7) [reply app/client/cypress/support/Pages/PropertyPane.ts]: done

**Review by Aishwarya-U-R — COMMENTED** [review]
Cypress looks good.

**Pull request merged**